### ISSUE TITLE
Rome 1.0 does not follow the W3C Feed Validator recommendations

### ISSUE THREAD

**Issue by PatrickGotthard** [rometools/rome#147]
Using the code

```java
package test;

import java.util.Date;
import java.util.List;

import com.rometools.rome.feed.synd.SyndCategory;
import com.rometools.rome.feed.synd.SyndCategoryImpl;
import com.rometools.rome.feed.synd.SyndFeed;
import com.rometools.rome.feed.synd.SyndFeedImpl;
import com.rometools.rome.io.SyndFeedOutput;

public class TestApplication {

    public static void main(final String[] args) throws Exception {

        final SyndCategory category = new SyndCategoryImpl();
        category.setName("Category");

        final SyndFeed feed = new SyndFeedImpl();
        feed.setFeedType("rss_2.0");
        feed.setTitle("title");
        feed.setDescription("description");
        feed.setLink("https://example.com");

        feed.setCategories(List.of(category));
        feed.setCopyright("copyright");
        feed.setLanguage("de");
        feed.setPublishedDate(new Date());

        final String xml = new SyndFeedOutput().outputString(feed);
        System.out.println(xml);

    }

}
```

Rome produces

```xml
<?xml version="1.0" encoding="UTF-8"?>
<rss xmlns:dc="http://purl.org/dc/elements/1.1/" version="2.0">
  <channel>
    <title>title</title>
    <link>https://example.com</link>
    <description>description</description>
    <language>de</language>
    <copyright>copyright</copyright>
    <pubDate>Wed, 29 Dec 2021 20:45:10 GMT</pubDate>
    <category>Category</category>
    <dc:subject>Category</dc:subject>
    <dc:date>2021-12-29T20:45:10Z</dc:date>
    <dc:language>de</dc:language>
    <dc:rights>copyright</dc:rights>
  </channel>
</rss>
```

Accoring to the W3C feed validator (https://validator.w3.org/feed/) the feed is valid but the interoperability could be improved by implementing the following recommendations:
- A channel should not include both category and dc:subject
- A channel should not include both pubDate and dc:date
- A channel should not include both language and dc:language
- A channel should not include both copyright and dc:rights

**Comment by PatrickGotthard** [comment]
I think we should make this configurable and don't use Dublin Core by default.

**Comment by PatrickGotthard** [comment]
I've just checked some variations of the original issue and found out that the mentioned Dublin Core entries are perfectly fine for all other feed types (it is only an RSS 2.0 "problem"). I think it is not possible to fix this **warning** (it's no error) without completely overahuling the DC module implementation.

Also I found out that there are many possibilities to generate invalid feeds - I think it needs a major rewrite to be fully standards-compliant.

Due to the lack of active maintainers, the age of the issue and the low severity, I'll close the issue.
